prow/cmd/deck/main.go

```diff
@@ -1327,9 +1336,10 @@ func canTriggerJob(user string, pj prowapi.ProwJob, cfg *prowapi.RerunAuthConfig
 // POST request, and the user has the necessary permissions. Otherwise, it writes the config
 // for a new job but does not trigger it.
 func handleRerun(prowJobClient prowv1.ProwJobInterface, createProwJob bool, cfg authCfgGetter, goa *githuboauth.Agent, ghc githuboauth.GitHubClientGetter, cli prowgithub.RerunClient, pluginAgent *plugins.ConfigAgent) http.HandlerFunc {
+	log := logrus.WithField("handler", "/rerun")
```

Comment:
Nit: Would it make sense to define the loggers at the router level and pass them in? That way:
* If someone changes the route for something, they are likely to also change the log accordingly
* If someone adds a new handler, they will hopefully also pass in a logger with a `handler` field

Reply:
Done.